firefox_ui_harness/default_prefs.py

```diff
@@ -36,6 +36,7 @@
     'extensions.showMismatchUI': False,
     'extensions.update.enabled': False,
     'extensions.update.notifyUser': False,
+    'focusmanager.testmode': True,
```

Comment:
With the addition of focusmanager.testmode: True, we should have recorded in default_prefs everything that we talked about needing and wanting to include from mozmill.

The bug you filed, with links to mozmill's preferences settings, is here: 
https://bugzilla.mozilla.org/show_bug.cgi?id=1123683

David wasn't sure when I asked why focusmanager.testmode wasn't included with the Marionette client. I've filed a bug and will submit a patch for Marionette. David said he's "interested to see how the webapi tests cope with that pref."